From e47066dd5173cd5dbd9812609855f1d164e99128 Mon Sep 17 00:00:00 2001
From: Paul Bolle <pebolle@tiscali.nl>
Date: Fri, 5 Sep 2014 13:09:25 +0200
Subject: [PATCH 062/173] gpio: fix 'CONFIG_GPIO_IRQCHIP' comments

commit 7d75a871888e3f5e1a7c99bf240d1cd67d8bdfa0 upstream.

These two typos were introduced in commit 1425052097b5 ("gpio: add IRQ
chip helpers in gpiolib").

The correct symbol name is CONFIG_GPIOLIB_IRQCHIP.

[jkosina@suse.cz: add changelog]
Signed-off-by: Paul Bolle <pebolle@tiscali.nl>
Signed-off-by: Jiri Kosina <jkosina@suse.cz>
Signed-off-by: Pengyu Ma <pengyu.ma@windriver.com>
---
 include/linux/gpio/driver.h |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/linux/gpio/driver.h b/include/linux/gpio/driver.h
index eb00032..f38f707 100644
--- a/include/linux/gpio/driver.h
+++ b/include/linux/gpio/driver.h
@@ -105,7 +105,7 @@ struct gpio_chip {
 
 #ifdef CONFIG_GPIOLIB_IRQCHIP
 	/*
-	 * With CONFIG_GPIO_IRQCHIP we get an irqchip inside the gpiolib
+	 * With CONFIG_GPIOLIB_IRQCHIP we get an irqchip inside the gpiolib
 	 * to handle IRQs for most practical cases.
 	 */
 	struct irq_chip		*irqchip;
@@ -218,7 +218,7 @@ int gpiochip_irqchip_add(struct gpio_chip *gpiochip,
 		irq_flow_handler_t handler,
 		unsigned int type);
 
-#endif /* CONFIG_GPIO_IRQCHIP */
+#endif /* CONFIG_GPIOLIB_IRQCHIP */
 
 struct gpio_desc *gpiochip_request_own_desc(struct gpio_chip *chip, u16 hwnum,
 					    const char *label);
-- 
1.7.5.4

